tests/plugins/widget/manual/dragwidget.md

```diff
@@ -0,0 +1,11 @@
+@bender-tags: widget, bug, 4.7.2, 711
+@bender-ui: collapsed
+@bender-ckeditor-plugins: widget, wysiwygarea, toolbar, sourcearea, image2, contextmenu
+
+**Scenario:**
+
+1. Click `Image` on toolbar.
+2. Input image URL, set width to 100 and mark `Captioned image` checkbox.
```

Comment:
IMO test should already include such image. We have dedicated asset for inserting images in tests: `lena.jpg`.